trulens_eval/trulens_eval/Leaderboard.py

```diff
@@ -28,6 +28,8 @@ def app():
     )
     df, feedback_col_names = lms.get_records_and_feedback([])
 
+    test_threshold = 0.5
```

Comment:
Our current implementation also used a threshold. Is there somewhere this constant can be moved so that it is shared among all downstream?

Reply:
Seems reasonable to move it to tru.py, but where else is the threshold used?

Reply:
or alternatively, as a constant in the Feedback() class?

Reply:
Not sure, but we had red/green text being chosen by something before, did we not?

Reply:
nvm, search `0.5` in  Evaluations.py 
Lets move both of those to `tru_feedback.py` and name the variable `default_pass_fail_color_threshold`